website/sidebars.js

```diff
@@ -11,31 +11,58 @@
 
 module.exports = {
   docs: [
-    'intro',
     {
       type: 'category',
-      label: 'Installation',
+      label: 'Introducing GitOps',
+      collapsed: false,
       link: {
         type: 'doc',
-        id: 'installation/index',
+        id: 'intro',
       },
       items: [
-        'installation/weave-gitops',
         {
           type: 'category',
-          label: 'Weave GitOps Enterprise',
+          label: 'Installation',
           link: {
             type: 'doc',
-            id: 'installation/weave-gitops-enterprise/index',
+            id: 'installation/index',
```

Comment:
Putting my comment here as the page in question isn't changed as part of this PR: The page's title should follow a similar pattern as the "part 1" and "part 2" pages and have a title preceded by "Part 0: ".

Reply:
it does and doesn't make sense to do that.

the installation section is still separate from the getting started section, the getting started just links back to installation in its first.

if i were to pull installation under getting started, then yes having it be part 0 makes sense (i can even 1-index more naturally there). but if we want installation to remain separate (and there is a good case for doing that), then it makes less sense to have titled with a number for people who are coming just to the installation bit

Reply:
![image (5)](https://user-images.githubusercontent.com/8898786/218076799-d111de1b-15a0-48c1-afb8-352f21dd182a.png)

Looks like this now